### PR TITLE
feat(design-system): modify typography component

### PR DESCRIPTION
**Pull request by tiborux**
EX-7638

### CODE REVIEW

File: packages/x-tailwindcss/src/x-tailwind-plugin/components/typography/title1/sizes.ts [hunk 1]
```diff
@@ -10,15 +10,15 @@ import { TailwindHelpers } from '../../../../types';
 export function titleSizes({ theme }: TailwindHelpers) {
   return {
     sm: {
-      fontSize: theme('fontSize.md')
+      fontSize: theme('fontSize.xl'),
+      lineHeight: theme('lineHeight.md')
     },
     md: {
       fontSize: theme('fontSize.xl'),
```
Comment (mnavarroespinosa) [inline packages/x-tailwindcss/src/x-tailwind-plugin/components/typography/title1/sizes.ts:17]: Check with the design. This should be `xl` or `32px` the values don't match.

File: packages/x-tailwindcss/src/x-tailwind-plugin/components/typography/title2/sizes.ts [hunk 2]
```diff
@@ -10,15 +10,15 @@ import { TailwindHelpers } from '../../../../types';
 export function titleSizes({ theme }: TailwindHelpers) {
   return {
     sm: {
-      fontSize: theme('fontSize.md'),
-      lineHeight: theme('lineHeight.md')
+      fontSize: theme('fontSize.xl')
```
Comment (mnavarroespinosa) [inline packages/x-tailwindcss/src/x-tailwind-plugin/components/typography/title2/sizes.ts:13]: Check with the design. This should be `xl` or `20px`, the values don't match.

**Review by mnavarroespinosa — CHANGES_REQUESTED** [review]

**Comment by mnavarroespinosa** [comment]
Is it intended that `Text 1` and `Text 2` are identical, isn't it? Just to double-check.

**Comment by ajperezbau** [comment]
The `font-weight` of the bold variants seem to be 700 in the designs, but it is set to 600 in the project.

**Review by ajperezbau — COMMENTED** [review]
Nothing more apart from the things that @mnavarroespinosa commented.
As a side note, I was talking with @javieri-empathy before, and he suggested that maybe in the designs only the tokens should be specified, to avoid confusions with the values and to ease the maintenance of the designs.

**Comment by tiborux** [comment]
> Nothing more apart from the things that @mnavarroespinosa commented. As a side note, I was talking with @javieri-empathy before, and he suggested that maybe in the designs only the tokens should be specified, to avoid confusions with the values and to ease the maintenance of the designs. It makes sense to me, we can comment it tomorrow with the team if you think alike.

I think it is better, I usually check the token, not the value...

**Comment by tiborux** [comment]
image

**Review by mnavarroespinosa — APPROVED** [review]
good job!

**Review by ajperezbau — APPROVED** [review]

**Pull request merged**